draft-ietf-quic-http.md

```diff
@@ -1941,7 +1941,9 @@ In HTTP/3, setting values are variable-length integers (6, 14, 30, or 62 bits
 long) rather than fixed-length 32-bit fields as in HTTP/2.  This will often
 produce a shorter encoding, but can produce a longer encoding for settings which
 use the full 32-bit space.  Settings ported from HTTP/2 might choose to redefine
-the format of their settings to avoid using the 62-bit encoding.
+their value to limit it to 30 bits for more efficient encoding, or to make use
+of the 62-bit space (for example, if encoding the original value often requires
+eight bytes anyway).
```

Comment:
This parenthetical feels kind of awkward; maybe just something like "or might choose to make use of the 62-bit space if more than 30 bits are required"?

Reply:
Thanks for the suggestion.